translations/fr/_layouts/sidebar.md

```diff
@@ -5,6 +5,7 @@
   - [Outils pour réussir](/translations/fr/04-outils-pour-reussir.md)
   - [Comment contribuer à l'open source](/translations/fr/05-comment-contribuer-à-l-open-source.md)
   - [La sauce secrète](/translations/fr/06-la-sauce-secrète.md)
-  - [Conclusion](/translations/fr/07-conclusion.md)
+  - [Les types de contributions](/translations/fr/07-les-types-de-contributions.md)
```

Comment:
IMHO, whatever change is related to French translation, there should be another PR.

Reply:
I get it @antonio-pedro99, but  @BekahHW requested to get those fixed right at the moment. In the future, we'll do this. :)

Reply:
@CBID2 I think you misunderstood. I _was_ looking at the wrong translation. It should be in a separate PR. I didn't realize that got merged in. I know I caught that in the French PR and since I don't speak either of these languages, I thought I was looking at the br translation. I also wonder if there were other issues that didn't get resolved in that French PR. That's another reason an issue should be made for that. This is one of the complicated parts of having translations when we don't have a maintainer team to support reviewing the PRs bc they don't speak the language.

Reply:
Hi @BekahHW @CBID2 @antonio-pedro99, 
- The Portuguese part is finished, let review it and merge it
- And have another PR for French, we don't have issues but fixes
- Then we can fix anything related to French translation (I don't have hand on this PR to edit them and submit, you all will notice that I was sending PR to Antonio's forked repo)

Reply:
Hey @BekahHW, thanks for clarifying this. @antonio-pedro99, I'll revert the commits that contain the changes in French. @geoffreylgv, I'll create a separate issue for the French PR and we can create it. Sound good?